tests/conftest.py

```diff
@@ -21,7 +21,18 @@
 sys.path.insert(0, str(PROJECT_ROOT / "src"))
 sys.path.insert(0, str(PROJECT_ROOT / "tests"))
 
-from d810._vendor.clang.cindex import Config, Cursor, CursorKind, Index, TranslationUnit
+# Try to import clang bindings - optional dependency for AST comparison
```

Comment:
Hmm, these should definitely be moved to the system tests if you're going to have this here.